BREAKING CHANGE: deprecating SI (SCC)

gary1998 opened the pull request:
<!--- See what makes a good Pull Request at : https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/.github/CONTRIBUTING.md --->

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

gary1998 commented:
Security Insights (formerly Security Advisor) is sun-setting. We've already taken our backend services down and we're not gonna provide any substitute for the current consumers. That's why, we're not gonna provide any deprecation message for now.

hkantare commented:
Since the service already deprecated and shutdown ..Can you just keep the docs and add a NOTE that these services are shutdown and will remove docuemntation in next release

gary1998 commented:
@hkantare I've added the docs back to the PR and I've added a deprecation message to them. Please have a look at it and let us know your comments on it.
